### ISSUE TITLE
Use with subscriptions

### ISSUE THREAD

**Issue by scf4** [apollographql/persistgraphql#33]
Does this work with the subscriptions interface? I see no mention in the docs.

**Comment by AlexLeung** [comment]
I would also like to know whether or not Subscriptions are supported.

**Comment by Poincare** [comment]
I don't believe there's a subscription-ready network interface that's available for immediate use but this shouldn't be hard to implement. It will essentially look like the [HTTP interface](https://github.com/apollographql/persistgraphql/blob/master/src/network_interface/ApolloNetworkInterface.ts#L67). Would either of you (@scf4, @AlexLeung) be interested in helping with this?
